ios/Podfile.lock

```diff
@@ -423,16 +423,16 @@ SPEC CHECKSUMS:
   abseil: 18063d773f5366ff8736a050fe035a28f635fd27
   AppAuth: 73574f3013a1e65b9601a3ddc8b3158cce68c09d
   BoringSSL-GRPC: db8764df3204ccea016e1c8dd15d9a9ad63ff318
-  cloud_firestore: 11c87f1d0b4193831045a320aed50723b67f2ea3
```

Comment:
Let's try and get to the bottom of this so we don't keep fighting over the checksums.  There's an explanation of why it happens and a potential solution here: 

https://artsy.github.io/blog/2016/05/03/podspec-checksums/

If you run `pod update` from `ios/` do you get the same checksums as currently in `dev`?

Reply:
I guess this was me too - I'll stop trying to blame other people 😄